css/style.css

```diff
@@ -0,0 +1,255 @@
+@font-face {
+  font-family: "Cocogoose";
+  src:
+    local("Cocogoose"),
+    url("../fonts/Cocogoose_trial.otf") format("opentype");
+}
+
+:root {
+  --primary-color: #ec5242;
+  --secondary-color: #272a31;
+  --tertiary-color: #d3d3d3;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  vertical-align: baseline;
+}
+
+html {
+  font-size: calc(15px + 0.390625vw);
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  overflow-x: hidden;
+}
+
+.logo {
+  width: 150px;
+  height: 50px;
+  margin-top: 10px;
+}
```

Comment:
Kindly ensure that the pages use the exact same colors for text, backgrounds, borders, etc, from the original design. All page elements should also have similar spacing, borders, shadows, and paddings than the orignal design.